ansible/veos_vtb

```diff
@@ -18,6 +18,7 @@ all:
           - t1-64-lag-clet
           - t1-backend
           - t1-smartswitch
+          - smartswitch-ha
```

Comment:
Maybe we should call it - t1-smart switch-ha

Reply:
Addressed.